force-app/main/default/classes/ContactTriggerHandler.cls

```diff
@@ -0,0 +1,34 @@
+public with sharing class ContactTriggerHandler {
+    
+    // When a contact is inserted
+	// if DummyJSON_Id__c is null, generate a random number between 0 and 100 and set this as the contact's DummyJSON_Id__c value
+	
+	public static void setJsonId(List<Contact> newContact) {
```

Comment:
These methods are well structured and the logic is sound, but the idea with the Trigger handler framework being used here is to leverage (override) the existing methods in TriggerHandler (`beforeInsert()`, `afterInsert()`, `afterUpdate()`)